docs/source/sections/progapi/progAPI-v4/Programmatic-Demo.rst

```diff
@@ -7,6 +7,11 @@ About
 
 There are two Application Programming Interfaces (APIs) for interacting with ISB-CGC hosted data. The first API, recommended for interacting with Big Query Tables is the `Google SDK <https://cloud.google.com/bigquery/docs/reference/rest/>`_. ISB-CGC also provides a `Swagger API <https://api-dot-isb-cgc.appspot.com/v4/swagger/>`_, intended for interacting with our webapp and data generated through our webapp via the command line.
 
+The ISB-CGC APIs can also be used via Python and R. We have tutorial notebooks available in our `Community Notebook Repository <https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/HowTos.html#i-m-a-novice-how-do-i>`_. 
+
+- `Python Notebook <https://github.com/isb-cgc/Community-Notebooks/blob/Staging-Notebooks/Notebooks/How_to_use_ISB_CGC_APIs.ipynb>`_
```

Comment:
The link is referencing Staging-Notebooks. Is that the highest level?

Reply:
Your right, I grabbed the wrong URL. It is fixed now. Thanks!